items/src/test/scala/weco/api/items/fixtures/ItemsApiFixture.scala

```diff
@@ -30,12 +33,28 @@ trait ItemsApiFixture extends SierraSourceFixture {
 
   def withItemsApi[R](
     catalogueResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
-    sierraResponses: Seq[(HttpRequest, HttpResponse)] = Seq()
+    sierraResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
+    contentApiVenueResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
+    time: Int
```

Comment:
It might be clearer in tests what's going on if this took a `Clock` and there was another `withClock` function to wrap creating that clock (i'd also provide a default here for tests that do not care about the time).